apiserver/ServeSubmission.md

```diff
@@ -0,0 +1,681 @@
+# Managing RayServe with the API server
+
+Ray Serve is a scalable model serving library for building online inference APIs. This document describes creation and management of Ray Serve enabled Ray clusters
+
+## Deploy KubeRay operator and API server
+
+Reffer to [readme](README.md) for setting up KubRay operator and API server.
+
+```shell
+make operator-image docker-image cluster load-operator-image load-image  deploy-operator deploy
+```
+
+Once they are set up, you first need to create a Ray cluster
+
+## Creating a cluster with RayServe support
+
+Before creating a cluster you need to create a template using the following command:
+
+```shell
+curl -X POST 'localhost:31888/apis/v1/namespaces/default/compute_templates' \
+--header 'Content-Type: application/json' \
+--data '{
+  "name": "default-template",
+  "namespace": "default",
+  "cpu": 2,
+  "memory": 4
+}'
+```
+
+Up until rescently the only way to create a Ray cluster supporting RayServe was by using `Create ray service` APIs. Although it does work, quite often you want to create cluster supporting Ray serve so that you can experiment with serve APIs directly. Now it is possible by adding the following annotation to the cluster:
```

Comment:
```suggestion
Up until recently the only way to create a Ray cluster supporting RayServe was by using `Create ray service` APIs. Although it does work, quite often you want to create cluster supporting Ray serve so that you can experiment with serve APIs directly. Now it is possible by adding the following annotation to the cluster:
```

Reply:
Done